lib/house.rb

```diff
@@ -0,0 +1,46 @@
+class House
+  attr_reader :price, :address, :rooms
+
+  def initialize(price, address)
+    @price = price
```

Comment:
be careful-- the interaction pattern wants price to be an integer, and yours is still the string.  Be sure your tests/implementation match the interaction pattern exactly.